assets/scripts/modules/AppData/GalleryData.js

```diff
@@ -0,0 +1,228 @@
+/* 
+** gallery data module holds array for all gallery pages 
+** the array holds objects with the following fields
+** topIcon: for the top icons for displayed main gallery item
+** galleryType: to specify where to classify the item into either "main" or "series" e.t.c
+** title: for setting the item title
+** poster: for the poster image rather backgrund image of the item
+*/
+const GalleryData = [

```

Comment:
Only modules should start with capital letters to differentiate them from ordinary functions. You should have seen this in the course I shared.